### PR TITLE
DTSPO-7157    Update tags

### PR DESCRIPTION
**Pull request by mokainos**

### CODE REVIEW

**Review by timja — COMMENTED** [review]
link to where you've tested this?

**Comment by mokainos** [comment]
> 
deployed this resource using this branch: 
https://portal.azure.com/#blade/Microsoft_Azure_Monitoring/UpdateVNextAlertRuleBlade/ruleInputs/%7B%22alertId%22%3A%22%2Fsubscriptions%2F3eec5bde-7feb-4566-bfb6-805df6e10b90%2FresourceGroups%2Fmi-test-rg%2Fproviders%2Fmicrosoft.insights%2Fscheduledqueryrules%2Fmi-server-errors3%22%7D

File: templates/Alert.json [hunk 1]
```diff
@@ -49,6 +49,15 @@
...
     "appInsightsQuery": {
       "type": "string"
+    },
+    "commonTags": {
+      "type": "string"
+    },
+    "additionalTags": {
+      "type": "object",
+      "defaultValue": {
+        "[concat('hidden-link:', resourceId('microsoft.insights/components', parameters('appInsightsName')))]": "Resource"
```
Comment (timja) [inline templates/Alert.json]: I think this should be in the variables section? is there a reason you made it a paremeter to the template?

**Comment by timja** [comment]
just double checking you've re-tested since moving it to variables?

File: templates/Alert.json (hunk 1 above)
Comment (mokainos) [reply templates/Alert.json]: Migrated to variable

**Review by timja — APPROVED** [review]
have you re-tested? code looks good

**Comment by mokainos** [comment]
> have you re-tested? code looks good

https://portal.azure.com/#blade/Microsoft_Azure_Monitoring/UpdateVNextAlertRuleBlade/ruleInputs/%7B%22alertId%22%3A%22%2Fsubscriptions%2F3eec5bde-7feb-4566-bfb6-805df6e10b90%2FresourceGroups%2Fmi-test-rg%2Fproviders%2Fmicrosoft.insights%2Fscheduledqueryrules%2Fmi-server-errors4%22%7D

**Pull request merged**